src/main/webapp/logged_in_homepage.html

```diff
@@ -1,109 +1,103 @@
+                <span aria-hidden="true">&times;</span>
+                </button>
+              </div>
+              <div class="modal-body">
+                <form action="/matches" id="new_match_form" method="POST">
+                  <select  name="new-matches" id="new-matches" multiple>
+                    <!-- populated using JavaScript -->
+                  </select>
+                  <br/><br/>
+                </form>
+              </div>
+              <div class="modal-footer">
+                <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                <button type="submit" form="new_match_form" class="btn btn-primary">Save Matches</button>
+              </div>
+            </div>
+          </div>
+        </div>
       </div>
-    </div>
-  </nav>
-
- <header class="masthead text-center text-white">
-    <div class="masthead-content">
+    </section>
+    <!-- Footer -->
+    <footer class="py-5 bg-black">
       <div class="container">
-        <h1 class="masthead-heading mb-0">Logged In Homepage</h1>
+        <p class="m-0 text-center text-white small">Copyright &copy; Techish 2020</p>
       </div>
-    </div>
-    <div class="bg-circle-1 bg-circle"></div>
-    <div class="bg-circle-2 bg-circle"></div>
-    <div class="bg-circle-3 bg-circle"></div>
-    <div class="bg-circle-4 bg-circle"></div>
-  </header>
-
-  <section>
-    <div class="container">
-
-    <h3>Match History:</h3>
-    <ul id="match-history"></ul>
-  
-
-    <div class="form" id="matchForm">
-      <form class="form-container">
-        <h3>Get New Matches</h3>
-
-        <label for="product_area"><b>Product Area</b></label>
-        <input
-          type="text"
-          placeholder="Enter Product Area"
-          id="product_area"
-          required
-        />
-        <input type="button" class="btn btn-primary" value="Submit" onclick="sendMatchRequest()" />
-      </form>
-    </div>
-
-  <hr>
-  <h3>New Matches (Going to be a popup):</h3>
-    <!-- FORM TO SELECT WHICH NEW MATCHES TO SAVE -->
-    <form action="/matches" id="new_match_form" method="POST">
-      <label for="new-matches">Select matches to save:</label>
-      <select class="selectpicker" name="new-matches" id="new-matches" multiple>
-        <!-- populated using JavaScript -->
-      </select>
-      <br/><br/>
-      <input type="submit" value="Submit" />
-    </form>
-
-    </div>
-  </section>
-
-
-  <!-- Footer -->
-  <footer class="py-5 bg-black">
-    <div class="container">
-      <p class="m-0 text-center text-white small">Copyright &copy; Techish 2020</p>
-    </div>
-    <!-- /.container -->
-  </footer>
-
-  <!-- Bootstrap core JavaScript -->
-  <script src="vendor/jquery/jquery.min.js"></script>
-  <script src="vendor/bootstrap/js/bootstrap.bundle.min.js"></script>
-
-
+      <!-- /.container -->
+    </footer>
+    <!-- Bootstrap core JavaScript -->
+    <script src="vendor/jquery/jquery.min.js"></script>
```

Comment:
Could you move these script includes to the top with the others?

Reply:
I might be wrong about this so let me know, but I thought that the JavaScript includes were supposed to go at the bottom of the .html?

Reply:
From what I understand, script tags can be included in either the <head> element or at the bottom of the <body> element. The advantage of placing it at the bottom is it improves the display speed (script interpretation slows down the display).